ui/app/components/date-dropdown.js

```diff
@@ -0,0 +1,38 @@
+import Component from '@glimmer/component';
```

Comment:
Make `add-on` component?

Reply:
I don't think it's quite generic enough yet. Keeping it as its own component is 👍 though